<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window2</class>
 <widget class="QWidget" name="window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>124</height>
               </size>
              </property>
              <property name="text">
               <string>选择节点：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sensorCB">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>124</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>一号节点</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>二号节点</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>三号节点</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>124</height>
               </size>
              </property>
              <property name="text">
               <string>选择数据：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="indexCB">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>124</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>温度</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>湿度</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>光照</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="chooseBT">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>124</height>
           </size>
          </property>
          <property name="text">
           <string>确定选择</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="title">
         <string>图表显示</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="chartGL"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>124</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>124</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image:url(:/back.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
